CountryRoad.java

```diff
@@ -0,0 +1,12 @@
+package JavaAutomationLab;
+
+public class CountryRoad extends RouteSection {
```

Comment:
let's remove that on future task - no need to make hierarchy here

Reply:
> let's remove that on future task - no need to make hierarchy here

Modified